CONTRIBUTING.md

```diff
@@ -67,7 +67,7 @@ presented.
    to ask questions.
 
 2. [Values](https://github.com/flutter/flutter/wiki/Values),
-   which talks about what we care most about.
+   which elucidates what we care most about.
```

Comment:
this one is harder to understand

Reply:
Ok I'll change that word to a simpler one!